Modules/Office365DSC/DSCResources/MSFT_SPOSiteDesignRights/readme.md

```diff
@@ -0,0 +1,45 @@
+# SPOSiteDesignRights
+
+## Description
+
+This resource configures rights on Site Designs.
+
+## Parameters
+
+    Ensure
+      - Required: No (Defaults to 'Present')
+      - Description: `Present` is the only value accepted.
+          Configurations using `Ensure = 'Absent'` will throw an Error!
```

Comment:
Is it? Looks like you are properly handling the Absent case.

Reply:
fixed